src/web/src/components/Banner.tsx

```diff
@@ -114,38 +114,33 @@ export default function Banner() {
     <>
       <div className={classes.heroBanner}>
         <BannerDynamicItems />
-
-        <Typography variant="h1" className={classes.h1}>
+        <LandingButtons />
+      </div>
+      <div className={classes.textsContainer}>
+        <Typography variant="h1" className={classes.telescopeTitle}>
           Telescope
         </Typography>
-
+        <Typography variant="h4" className={classes.quoteText}>
+          “I think one of my proudest contributions to date was for Node.js.
```

Comment:
We'll have to think about how to provide these quotes.  Do we embed an array of them into the front-end and choose at random?  Or do we have the "image" service do it, since it's similar?

Reply:
I wonder if we can have a text file that people can add quotes to it through a pull request like what npm does https://github.com/npm/npm-expansions, and we select and random quotes from it.

Reply:
I think having the image service do this is a good idea